frontend/src/pages/application/Pipelines.vue

```diff
@@ -50,7 +50,7 @@
             @stage-deleted="(stageIndex) => stageDeleted(pipeline, stageIndex)"
         />
     </div>
-    <EmptyState v-else :featureUnavailable="!featureEnabled">
+    <EmptyState v-else :featureUnavailable="!featureEnabled" :featureUnavailableMessage="'This feature is not available'">
```

Comment:
I'd be tempted to move this to a default in `EmptyState`

Reply:
https://github.com/FlowFuse/flowfuse/blob/12af24aee95be79c1702ba5a401e4eb02eeba598/frontend/src/components/banners/FeatureUnavailable.vue#L23-L26

Reply:
It's being explicitly set to `null` (rather than `undefined`) in EmptyState, and thus the default is not applying.